.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "makefile.extensionOutputFolder": "./.vscode"
+}
```

Comment:
should be newlines at the EOF, I think if you run the prettier command it will fix those